app/pages/admin/organization-status.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import moment from 'moment';
+
+import ProjectIcon from '../../components/project-icon';
+import LoadingIndicator from '../../components/loading-indicator';
+
+import Toggle from './project-status/toggle';
+
+class OrganizationStatus extends Component {
+  constructor(props) {
+    super(props);
+    this.forceUpdate = this.forceUpdate.bind(this);
```

Comment:
@simoneduca from what I can tell from the [json-api-client README](/zooniverse/json-api-client/blob/master/README.md) that listener you linked too will fire when `organisation` changes. So the component renders when the organisation resource is changed inside or outside this component. I think.